specs/Spec.GF128.fst

```diff
@@ -0,0 +1,72 @@
+module Spec.GF128
+
+open FStar.Mul
+open Lib.IntTypes
+open Lib.Sequence
+open Lib.ByteSequence
+
+open Spec.GaloisField
+
+//GF128 Field: to load bits in little-endian order set the parameters as follows
+//let load_elem (b:lbytes 16) : elem = load_felem_le #gf128 b
+//let store_elem (e:elem) : lbytes 16 = store_felem_le #gf128 e
+//and use `gf128_le/fmul` instead of `gf128/fmul_be`
+
+let irred = mk_int #U128 #SEC 0xE1000000000000000000000000000000
+let gf128 = gf U128 irred
+let elem = felem gf128
+let load_elem (b:lbytes 16) : elem = load_felem_be #gf128 b
+let store_elem (e:elem) : lbytes 16 = store_felem_be #gf128 e
+
+let irred_le = mk_int #U128 #SEC 0x87
```

Comment:
what does `le`mean? lesser or equal? comments please

Reply:
`le` here stands for little-endian mode. I will adds a comment that clears things up.